Control.cpp

```diff
@@ -0,0 +1,492 @@
+    digitalWrite(kMotorLeftBack1, LOW);
+    analogWrite(kMotorLeftBack2, pwm_left);
+    digitalWrite(kMotorRightForward1, LOW);
+    analogWrite(kMotorRightForward2, pwm_right);
+    digitalWrite(kMotorRightBack1, LOW);
+    analogWrite(kMotorRightBack2, pwm_right);
+}
+
+void Control::moveBack() {
+    digitalWrite(kMotorLeftForward1, LOW);
+    analogWrite(kMotorLeftForward2, kPwm_max);
+    digitalWrite(kMotorLeftBack2, LOW);
+    analogWrite(kMotorLeftBack1, kPwm_max);
+    digitalWrite(kMotorRightForward2, LOW);
+    analogWrite(kMotorRightForward1, kPwm_max);
+    digitalWrite(kMotorRightBack2, LOW);
+    analogWrite(kMotorRightBack1, kPwm_max);
+}
+
+void Control::writeNumLCD(const int num) {
+  lcd.clear();
+  lcd.print(num);
+}
+
+void Control::writeLyricsLCD(const char letra) {
+  lcd.print(letra);
+}
+
+void Control::writeLCD(const String sE1, const String sE2) {
+  lcd.clear();
+  lcd.print(sE1);
+  lcd.setCursor(0, 1);
+  lcd.print(sE2);
+}
+
+void Control::writeLCDdown(const String sE1) {
+  lcd.setCursor(0, 1);
+  lcd.print(sE1);
+}
+
+void Control::printLocation(double x, double y, double z) {
+  lcd.clear();
+  lcd.setCursor(0,0);
+  lcd.print("X:");
+  lcd.setCursor(2,0);
+  lcd.print(x);
+
+   lcd.setCursor(7,0);
+  lcd.print("Y:");
+  lcd.setCursor(9,0);
+  lcd.print(y);
+
+  lcd.setCursor(11,0);
+  lcd.print("Z:");
+  lcd.setCursor(13,0);
+  lcd.print(z);
+
+delay(kTimeSeeLocation);
+}
+
+void Control::setup() {
+  dispenser.attach(42);
+  dispenser.write(90);   // Initialize the dispenser in 90 degrees(in the middle).
+}
+
+void Control::drop_one_kit_left() {
+  dispenser.write(dispenserDegrees_0);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_100);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_90);
+  delay(kTime_2ms);
+}
+
+void Control::drop_one_kit_right() {
+  dispenser.write(dispenserDegrees_180);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_80);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_90);
+  delay(kTime_2ms);
+}
+
+void Control::drop_two_kits_right() {
+  dispenser.write(dispenserDegrees_180);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_80);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_90);
+  delay(kTime_2ms);
+  dispenser.write(dispenserDegrees_180);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_80);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_90);
+  delay(kTime_2ms);
+}
+
+void Control::drop_two_kits_left() {
+  dispenser.write(dispenserDegrees_0);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_100);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_90);
+  delay(kTime_2ms);
+  dispenser.write(dispenserDegrees_0);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_100);
+  delay(kTime_1sec);
+  dispenser.write(dispenserDegrees_90);
+  delay(kTime_2ms);
+}
+
+bool Control::heatVictim(){
+  
+}
+bool Control::visualVictim(){
+  
+}
+bool Control::colorVictim(){
+  
+}
```

Comment:
Do something in here

Reply:
I already have the heatVictim function, I will do the other functions with the vision person.